### PR TITLE
update sponsors logo band 'neds'

### PR DESCRIPTION
**Pull request by infernalslam**

### CODE REVIEW

**Comment by phanan** [comment]
Just curious, what's wrong with the current logo?

**Comment by infernalslam** [comment]
@phanan 
image

File: package.json [hunk 1]
```diff
@@ -29,4 +29,4 @@
     "hoek": "^5.0.3",
     "request": "^2.85.0"
   }
-}
+}
```
Comment (phanan) [inline package.json]: Please add a new line.

Comment (infernalslam) [reply package.json]: ok

Comment (infernalslam) [reply package.json]: I get it

**Comment by phanan** [comment]
Thanks!

**Pull request merged**